docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst

```diff
@@ -0,0 +1,193 @@
+.. {#openvino_docs_install_guides_installing_openvino_zypper}
+
+Install OpenVINO™ Runtime on Linux From ZYPPER Repository
+=========================================================
+
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Linux operating
+                 system, using the ZYPPER repository.
+
+.. note::
+
+   Note that the ZYPPER distribution:
+
+   * offers both C/C++ APIs
+   * does not offer support for NPU inference
+   * is dedicated to Linux users only
+   * additionally includes code samples
+
+.. tab-set::
+
+   .. tab-item:: System Requirements
+      :sync: system-requirements
+
+      | Full requirement listing is available in:
+      | :doc:`System Requirements Page <../../../about-openvino/release-notes-openvino/system-requirements>`
+
+      .. note::
+
+         OpenVINO RPM packages are compatible with and can be run on the following operating systems:
+
+         - openSUSE Tumbleweed
+
+   .. tab-item:: Processor Notes
+      :sync: processor-notes
+
+      | To see if your processor includes the integrated graphics technology and supports iGPU inference, refer to:
+      | `Product Specifications <https://ark.intel.com/>`__
+
+   .. tab-item:: Software
+      :sync: software
+
+      * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+      * GCC 8.2.0
+      * `Python 3.8 - 3.11, 64-bit <https://www.python.org/downloads/>`_
+
+
+Install OpenVINO Runtime
+########################
+
+Step 1: Set Up the Repository
++++++++++++++++++++++++++++++
+
+
+1. Create a ZYPPER repository file with the command below:
+
+   .. code-block:: sh
+
+      sudo zypper addrepo https://download.opensuse.org/repositories/science/openSUSE_Tumbleweed/science.repo
+      sudo zypper refresh
+
+2. Verify that the new repository is set up properly.
+
+   .. code-block:: sh
+
+      zypper lr |grep -i science
+
+   You will see the available list of packages.
+
+   To list available OpenVINO packages, use the following command:
+
+   .. code-block:: sh
+
+   zypper se openvino
+
+Step 2: Install OpenVINO Runtime Using the YUM Package Manager
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Install OpenVINO Runtime
+-------------------------
+
+.. tab-set::
+
+   .. tab-item:: The Latest Version
+      :sync: latest-version
+
+      Run the following command:
+
+      .. code-block:: sh
+
+         sudo zypper install openvino
+
+Check for Installed Packages and Version
+-----------------------------------------
+
+
+Run the following command:
+
+.. code-block:: sh
+
+   zypper se -i openvino
+
+.. note::
+
+   You can additionally install Python API using one of the alternative methods (:doc:`conda <openvino_docs_install_guides_installing_openvino_conda>` or :doc:`pip <openvino_docs_install_guides_installing_openvino_pip>`).
```

Comment:
as I can see `python3-openvino` is available as RPM package via zypper
so, we need to recommend alternative ways? Suppose no

Reply:
> as I can see `python3-openvino` is available as RPM package via zypper so, we need to recommend alternative ways? Suppose no
> 
> Secondly, I've tried to install `python3-openvino` and I see: ![image](https://private-user-images.githubusercontent.com/2566854/325350320-720c8d8f-ee9f-4414-a521-b70b176e002c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwMjEzNDAsIm5iZiI6MTcxNDAyMTA0MCwicGF0aCI6Ii8yNTY2ODU0LzMyNTM1MDMyMC03MjBjOGQ4Zi1lZTlmLTQ0MTQtYTUyMS1iNzBiMTc2ZTAwMmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MjVUMDQ1NzIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Njk3YzdkYmQ2ZjMxMTJkZWJmMzIxODBjMmNjM2Q3MTU1MDc5MTgyNjlhYWRhZGJmOGM3MzFkZmZlNmQyNDU0MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lEwazukAJ-lZYd2mdeQ6qMoHvZ28vhyJGeZ_CuEbJNA)

Let's not do this now. I'm working on the conventions and doing a lot of testing on many hardware platforms.

We are not going to do anything now in this Pull request. How are you @ilya-lavrenov?

As soon as we finish the approval of the Factory phase, I will insert new suggestions here, okay?